plugins/wdc/wdc-nvme.c

```diff
@@ -523,6 +523,16 @@ typedef enum
     EOL_RRER                = 108,	/* Raw Read Error Rate */
 } EOL_LOG_PAGE_C0_OFFSETS;
 
+static __u16 nvme_wdc_feat_buf_len[0x100] = {
```

Comment:
Could we use the version of this structure defined in nvme.c instead of defining our own version and having dual maintenance?  The nvme_feat_buf_len would need to be moved to a common include.

Reply:
I agree that would be clearer, but putting it in any of the common includes makes the compiler complain about the defined but not used value in any file that includes it.
I could set up a special header file containing just that nvme_feat_buf_len and only include it in the files that use it, but that might be adding more complexity than simply duplicating the values.

Reply:
Yea, I don't think it makes to create a special include for this.  Just thought it would nice have one a common struct vs having to maintain 2 versions of it.